compiler/ballerina-lang/src/main/java/org/ballerinalang/project/Project.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.project;
+
+import org.ballerinalang.compiler.JarResolver;
+import org.ballerinalang.model.elements.PackageID;
+import org.wso2.ballerinalang.compiler.util.CompilerContext;
+
+import java.nio.file.Path;
+
+/**
+ * Defines functionality of the Project.
+ *
+ * @since 2.0.0
+ */
+public interface Project {
+
+    CompilerContext.Key<Project> PROJECT_KEY = new CompilerContext.Key<>();
+
+    /**
+     * Return true is module exists in the project.
```

Comment:
"Return true is module exists in the project." has to be changed as "Returns true if the module exists in the project."